DNN Platform/DotNetNuke.Abstractions/Portals/IPortalSettings.cs

```diff
@@ -2,18 +2,121 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See LICENSE file in the project root for full license information.
 // 
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
This change is incorrect as this is the proper license attribute.  The lines above this should be removed